external-import/comlaude/src/main.py

```diff
@@ -87,6 +88,12 @@ def _generate_dynamic_custom_properties(helper, domain_object, score):
     return domain_name, custom_properties
 
 
+AUTHOR_NAME = "Comlaude"
+
+
+author_identity = Identity(name=AUTHOR_NAME, identity_class="organization")
```

Comment:
If possible, add the generate_id to have predictive ids and limit certain errors
from pycti import Identity
...
stix2.Identity(  
    id=Identity.generate_id(name, "organization"),                // pycti -> Identity  
    name=name, 
    identity_class="organization", 
)

Reply:
Hello Magafredo, Yes i think it's possible i will add the generate_id to have predictive ids